### ISSUE TITLE
[Feature Request] Can We Have @if and @for snippets?

### ISSUE THREAD

**Issue by marfrede** [johnpapa/vscode-angular-snippets#150]
Since angular 17 we have `@defer`, `@for`, `@if`, `@let` and `@switch` blocks.\
Would be nice to have snippets for those :-)

**Comment by saiyaff** [comment]
Yes, this is the much needed feature now. Surprised that the extension doesn't provide the support for the new control flow syntax yet

**Comment by johnpapa** [comment]
Thank you for this feature request and for the extra context.

Implemented in PR #187:
- `a-if` for `@if`
- `a-for` for `@for`

I’m closing this as addressed. If you want follow-up snippets for `@switch`, `@defer`, or `@let`, please open a dedicated issue and thank you again!
